### ISSUE TITLE
K8s launcher does not parse correctly parse remote configuration files

### ISSUE THREAD

**Issue by wikiselev** [nextflow-io/nextflow#738]
In one of our pipelines we have a parameter (`params.genomes`) which is not initialised but [used](https://github.com/cellgeni/rnaseq-noqc/blob/0ed143a537b710df89652ec2208e8ddf65ecdefe/main.nf#L95) in the `main.nf` . This parameter is instead initialised in a [config file](https://github.com/cellgeni/rnaseq-noqc/blob/master/conf/genomes.config) which is loaded in a [profile](https://github.com/cellgeni/rnaseq-noqc/blob/0ed143a537b710df89652ec2208e8ddf65ecdefe/nextflow.config#L34). This setting worked well before ver. 0.30.0 release. Now when I try to run the same pipeline with the same invocation I get this:
```
WARN: Access to undefined parameter `genomes` -- Initialise it to a default value eg. `params.genomes = some_value`
ERROR ~ Cannot get property 'GRCh38' on null object
```

This is all happening on k8s cluster.

**Comment by wikiselev** [comment]
`nf-core` pipelines may also be affected, since they also don't initialise `params.genomes` in their `main.nf`.

**Comment by pditommaso** [comment]
How is the run cmd line you are using?

**Comment by wikiselev** [comment]
`nextflow kuberun cellgeni/rnaseq-noqc -v testpvc:/mnt/gluster -profile docker -with-report report.html`

**Comment by wikiselev** [comment]
Hi @ewels, thanks for letting me know! I've checked it on our LSF cluster with `farm3` profile which does not affect the `params.genomes` parameter and it also works ok. @pditommaso can it be something k8s specific?

**Comment by pditommaso** [comment]
@wikiselev Could you please check if version `0.30.1-SNAPSHOT` solve the problem, eg: 

```
NXF_VER=0.30.1-SNAPSHOT nextflow kuberun .. etc
```

**Comment by pditommaso** [comment]
You this this [TODO](https://github.com/nextflow-io/nextflow/blob/1077a2dbd14a3779541ce1aee51a9226162956cf/src/main/groovy/nextflow/scm/AssetManager.groovy#L434-L437

What's happening is that `includeConfig` is not parsed when using `kuberun` command. This never worked, os it's a bit strange that you found that it was working with a previous NF version. 

Workaround, launch a pod with `nextflow kuberun login -v pvc:/etc`, then run the pipeline using `nextflow run cellgeni/rnaseq-noqc`. 

However there's also another issue in version 0.30.0, therefore use  `NXF_VER=0.30.1-SNAPSHOT` of a previous version.

**Comment by wikiselev** [comment]
Great, thanks for sorting this out, and for the workaround!

**Comment by pditommaso** [comment]
Actually I've just uploaded version `0.30.1`.

**Comment by wikiselev** [comment]
So, I can do `export NXF_VER=0.30.1`?

**Comment by pditommaso** [comment]
Or just update `nextflow -self-update`

**Comment by pditommaso** [comment]
I've uploaded a new snapshot that should solve this problem.

```
NXF_VER=0.30.2-SNAPSHOT nextflow kuberun .. etc
```

**Comment by pditommaso** [comment]
Available in `0.30.2`.
